bin/tachyon

```diff
@@ -75,9 +75,9 @@ TACHYON_LIBEXEC_DIR=${TACHYON_LIBEXEC_DIR:-$DEFAULT_LIBEXEC_DIR}
 . $TACHYON_LIBEXEC_DIR/tachyon-config.sh
 
 function runTest {
-  Usage="Usage: tachyon runTest <Basic|BasicNonByteBuffer|BasicRawTable> <STORE|NO_STORE>"
+  Usage="Usage: tachyon runTest <Basic|BasicNonByteBuffer|BasicRawTable> <STORE|NO_STORE> <SYNC_PERSIST|NO_PERSIST>"
 
-  if [ "$#" -ne 2 ]; then
+  if [ "$#" -ne 3 ]; then
```

Comment:
I manually cleaned up style inconsistencies in this file. I prefer to come up with a documentation on shell script style, and I'll write an auto formatter for all our shell scripts, don't want to manually format again...